clang/include/clang/Format/Format.h

```diff
@@ -5283,6 +5299,7 @@ struct FormatStyle {
   bool operator==(const FormatStyle &R) const {
     return AccessModifierOffset == R.AccessModifierOffset &&
            AlignAfterOpenBracket == R.AlignAfterOpenBracket &&
+           AlignAfterControlStatement == R.AlignAfterControlStatement &&
```

Comment:
Could you move this one line up? :)

Reply:
done, good catch, missed this when I changed the option name.